tools/install-powershell.readme.md

```diff
@@ -0,0 +1,56 @@
+
+## Features of install-powershell.sh
```

Comment:
@TravisEz13 - I have tried to run the markdown syntax test locally.  I have installed node-js and gulp as per the instructions in the script you linked, but it does not recognize 'test-mdsyntax' and I don't find it in the repo.  I'm on Windows 10.